linera-core/src/worker.rs

```diff
@@ -263,8 +263,7 @@ where
     storage: StorageClient,
     /// Configuration options for the [`ChainWorker`]s.
     chain_worker_config: ChainWorkerConfig,
-    recent_confirmed_value_cache: Arc<ValueCache<CryptoHash, Hashed<ConfirmedBlock>>>,
-    recent_validated_value_cache: Arc<ValueCache<CryptoHash, Hashed<ValidatedBlock>>>,
+    recent_executed_block_cache: Arc<ValueCache<CryptoHash, Hashed<ExecutedBlock>>>,
```

Comment:
Or maybe just `executed_block_cache`? A cache usually contains the most recent values anyway.